### PR TITLE
CBL-5752: Lazy Vectors

### PR DESCRIPTION
**Pull request by bmeike**
Just a stub QueryIndex class

### CODE REVIEW

**Review by pasin — APPROVED** [review]

File: common/main/java/com/couchbase/lite/Defaults.java [hunk 1]
```diff
@@ -26,8 +26,6 @@ private LogFile() {}
 
         /** Plaintext is not used, and instead binary encoding is used in log files */
         public static final boolean USE_PLAINTEXT = false;
-        @Deprecated
-        public static final boolean USE_PLAIN_TEXT = USE_PLAINTEXT;
```
Comment (borrrden) [inline common/main/java/com/couchbase/lite/Defaults.java:30]: Where did this go?

Comment (bmeike) [reply common/main/java/com/couchbase/lite/Defaults.java:30]: It is not in EE... Should it be?

Comment (borrrden) [reply common/main/java/com/couchbase/lite/Defaults.java:30]: When did it get introduced here?  Has any release gone out with this deprecated constant?

Comment (bmeike) [reply common/main/java/com/couchbase/lite/Defaults.java:30]: It doesn't cost anything to leave it there.  I put it back.

**Pull request merged**